<launch>
  <!-- caffe deployment mode -->
  <node name="caffe_deployment" pkg="cnn_localization" type="caffe_deployment" output="screen">
    <rosparam command="load" file="$(find cnn_localization)/config/caffe_config.yaml" />
  </node>
</launch>
